### PR TITLE
frontend: header size for shared organisation

### LINKED ISSUE #641: Adjust header size in SONAR shared home pages [2]
- the SONAR part of the header should be shorter that the part related to the organisation
- the search field of the organisation should be longer
- see images (before and after)

BEFORE:

Screenshot 2021-09-06 at 13 06 25

AFTER:

Screenshot 2021-09-06 at 13 06 25 copy

### PR DESCRIPTION
**Pull request by Garfield-fr**
* Closes #641.

Co-Authored-by: Bertrand Zuchuat <bertrand.zuchuat@rero.ch>

## Why are you opening this PR?

- Which task/US does it implement?
- Which issue does it fix?

## How to test?

- What command should I have to run to test your PR?
- What should I test through the UI?

## Code review check list

- [ ] Commit message template compliance.
- [ ] Commit message without typos.
- [ ] File names.
- [ ] Functions names.
- [ ] Functions docstrings.
- [ ] Unnecessary commited files?
- [ ] Extracted translations?

### CODE REVIEW

File: sonar/ext.py [hunk 1]
```diff
@@ -114,10 +114,11 @@ def init_views(self, app):
         """Initialize the main flask views."""
         app.url_map.converters['org_code'] = OrganisationCodeConverter
 
+        @app.route('/')
         @app.route('/<org_code:view>')
         def index(view):
             """Homepage."""
-            return render_template('sonar/frontpage.html')
```
Comment (jma) [inline sonar/ext.py:120]: Do you need to put a default view?

File: sonar/theme/templates/sonar/partial/navbar.html [hunk 2]
```diff
@@ -18,7 +18,8 @@
 <nav class="navbar navbar-expand-lg bg-{{ 'header' if page == 'home' else 'organisation' }}">
   <div class="container">
     {% if page != 'home' %}
-    <a class="navbar-brand" href="{{ url_for('index', view=view_code) }}">
+    {% set url = url_for('index') if (not view or view == config.SONAR_APP_DEFAULT_ORGANISATION) else url_for('index', view=view_code) %}
```
Comment (jma) [inline sonar/theme/templates/sonar/partial/navbar.html:21]: the default org code can be set in the python code.

File: sonar/theme/templates/sonar/partial/navbar.html [hunk 3]
```diff
@@ -29,11 +30,10 @@
       {% endif %}
       {% else %}
       <img src="{{ url_for('static', filename='images/global-logo.svg') }}"
-        height="{{ '30' if g.get('organisation') else '50' }}"
-        class="d-inline-block align-top {{ '' if g.get('organisation') else 'my-2' }}" alt="{{ _('SONAR logo') }}">
+        height="{{ '30' if g.get('organisation') and page == 'home' else '50' }}"
```
Comment (jma) [inline sonar/theme/templates/sonar/partial/navbar.html:33]: It would be better with a css class.

**Review by jma — APPROVED** [review]

**Comment by pronguen** [comment]
- [ ] I cannot open anymore an organisation view, whether it is shared or dedicated. Examples: https://sonardev.test.rero.ch/rero/ or https://sonardev.test.rero.ch/hepbejune/

**Review by pronguen — APPROVED** [review]

**Review by mmo — APPROVED** [review]

**Pull request merged**